### PR TITLE
204 make release 040

### PR DESCRIPTION
**Pull request by andped10**
Copied workflows from `Reflectometry` 

Changed how the module version is determined

### CODE REVIEW

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -14,50 +18,24 @@ jobs:
...
-        pip install dist/*.whl
+        pip install -e '.[dev]'
+        python -m build
     - name: Test with pytest
       run: |
-        pytest
+        cd ./dist
+        pytest ../
     - uses: actions/upload-artifact@v2
       with:
```
Comment (rozyczko) [inline .github/workflows/python-package.yml:39]: upload-artifact is up to ver.4 now but requires `overwrite: true` to work as before

File: .github/workflows/python-package.yml [hunk 2]
```diff
@@ -14,50 +18,24 @@ jobs:
...
-        name: easyCore - Python ${{ matrix.python-version }}
+        name: EasyCore - Python ${{ matrix.python-version }}
```
Comment (rozyczko) [inline .github/workflows/python-package.yml:40]: Why the uppercase here?

File: .github/workflows/python-publish.yml [hunk 3]
```diff
@@ -1,36 +1,37 @@
-# This workflows will upload a Python Package using Twine when a release is created
+# This workflow will
+# - build distribution package, pure python wheel
+# - deploy produced distribution package to PyIP
+#
 # For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
 
-name: Upload Python Package
+name: Deploy Python Package
```
Comment (rozyczko) [inline .github/workflows/python-publish.yml:7]: I guess `Deploy` might not be the right term, but also `Upload` is not fully describing the action.
Maybe `Publish`, after the action name?

**Review by rozyczko — APPROVED** [review]
Minor comments.
Please consider updating one of the actions used (upload-artifact)

File: .github/workflows/python-package.yml (hunk 1 above)
Comment (andped10) [reply .github/workflows/python-package.yml:39]: changed

File: .github/workflows/python-package.yml (hunk 2 above)
Comment (andped10) [reply .github/workflows/python-package.yml]: changed

File: .github/workflows/python-publish.yml (hunk 3 above)
Comment (andped10) [reply .github/workflows/python-publish.yml]: changed

**Pull request merged**